index.html

```diff
@@ -1429,6 +1429,8 @@ <h2>Substantive changes since <a href="https://www.w3.org/TR/2017/REC-dpub-aam-1
 							>Digital Publishing Accessibility API Mappings 1.0</a></h2>
 					<!-- EdNote: After each WD publish, move contents of this list into the next one below.  -->
 					<ul>
+						<li>4-Jan-2022: Fix incorrect doc-pagebreak role mentioned in ATK/AT-SPI mapping for doc-preface.</li>
```

Comment:
typo in the change log dates.  2023 now :)

Reply:
D'oh! Thanks for the catch!